backport 4.3 local repos changes and configuration of default behavior (#1902) (#1905)
uyuni-project/uyuni-docs: modules/client-configuration/pages/repositories.adoc

====

Local repositories are disabled during onboarding.

For Salt clients, local repositories will be disabled each time a channel state is executed.

> For example when applying the highstate or performing a package action.

Author: ```suggestion
For example, when applying the highstate or performing a package action.
```

> In case local repositories should stay enabled after onboarding the following pillar should be set for the affected Salt client:

Author: ```suggestion
If local repositories should stay enabled after onboarding the following pillar must be set for the affected Salt client:
```
